Convert/src/Controller/Conversion.java

```diff
@@ -0,0 +1,116 @@
+package Controller;
+
+import java.io.File;
+import java.util.ArrayList;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import Model.*;
+
+public class Conversion {
+
+	private SubUnit basePrefixUnit;
+	private SubUnit baseMesure;
+	private ArrayList<Unit> unitsList;
+	
+	public Conversion(String fileName) throws Exception{
+		super();
+		this.loadConfigFile(fileName);
+	}
+	
+	private void loadConfigFile(String fileName) throws Exception {
+		File xmlFile = new File(fileName);
+		
+		this.unitsList = new ArrayList<>();
+
+		DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();

```

Comment:
il existe des libs pour parser du xml; il faut les utiliser

Reply:
Tu penses à quelle librairie? Car DocumentBuilderFoactory fait partie de la librairie javax.xml.parser mais je l'utilise peut-être mal

Reply:
jackson par exemple : d'un objet java il peut te générer un xml et vice versa. le mapping se fait avec une simple méthode de read et s'il y a un souci dans le xml, genre des propriétés qui n'existe pas dans la classe java correspondante, il lèvera une erreur

https://www.baeldung.com/jackson-xml-serialization-and-deserialization